src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Todo.css';
+
+function Todo(props) {
+    return(
+        <p>{props.listItem.item}</p>
```

Comment:
What if here you had a ternary on a className for p referencing the list item's completed field? Also, it appears this isn't clickable.